lib/providers/mixer.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+exports = module.exports = function (options) {
+	options = options || {};
+	var uri = options.uri || 'https://mixer.com/';
+	var user = options.uri ? options.uri + 'api/v1/users/current' : 'https://mixer.com/api/v1/users/current';
+
+	return {
+		protocol: 'oauth2',
+		auth: uri + 'oauth/authorize',
+		token: uri + 'api/v1/oauth/token',
+		scope: ['user:details:self'],
+		useParamsAuth: true,
+		scopeSeparator: ',',
```

Comment:
The default scope is set to `user:details:self`, so there's not multiple scopes to separate. However if a consumer overrides this and requests additional scopes, they'll get rejected.